### PR TITLE
Round off buttons 'pill style' like on the website

### PR DESCRIPTION
**Pull request by jancborchardt**
Just like @Espina2 did on the website and also suggested for the app some time ago, so all credit to him! :tada:

This works very well with our form language, with the logo, and especially with the upcoming font change to Nunito: https://github.com/nextcloud/server/pull/11932

screenshot from 2018-10-19 18-55-55
screenshot from 2018-10-19 18-55-31

screenshot from 2018-10-19 16-12-11
buttons apps
screenshot from 2018-10-19 16-07-09
(That’s all the screenshots for now cause I ran into a maintenance mode I can’t recover from …)

Please review @nextcloud/designers and let me know if you find any non-pilled buttons.

### CODE REVIEW

**Comment by MariusBluem** [comment]
Am a bit sceptic about this, since text boxes are not in this pill design ...

**Comment by skjnldsv** [comment]
There is some css magic to do for the rounded white borders (see https://user-images.githubusercontent.com/925062/47232684-a3d36a80-d3d0-11e8-9cc9-f6807d545c56.png) It looks not ok with the antialiasing :)

Otherwise I really like it. :heart: 

Though it will be indeed a bit odd with the inputs.
@jancborchardt can you provide a screenshot of an `input[text] ` joined to a `submit.icon-confirm`? :)

**Comment by jancborchardt** [comment]
> Am a bit sceptic about this, since text boxes are not in this pill design ...

@MariusBluem yeah, I was also wondering about this. The reason I didn’t do the inputs yet is because traditionally only the search field is rounded like that. I would keep the inputs as they are for now, we can still adjust. (But even on our website I find it odd that the input fields are pill-style.)

> There is some css magic to do for the rounded white borders (see https://user-images.githubusercontent.com/925062/47232684-a3d36a80-d3d0-11e8-9cc9-f6807d545c56.png) It looks not ok with the antialiasing :)

Yeah – I think a big factor of this is cause that specific screenshot was made with 150% zoom. It looks better normally.

> Though it will be indeed a bit odd with the inputs.
> @jancborchardt can you provide a screenshot of an input[text] joined to a submit.icon-confirm? :)

Sorry, my installation is busted atm because of a maintenance mode issue, can’t do more screenshots right now. :\ (But for the joined inputs nothing should change – the icon-confirm should be floating on the right of the input, without background so you don’t even see it’s round.)

**Comment by skjnldsv** [comment]
> Sorry, my installation is busted atm because of a maintenance mode issue, can’t do more screenshots right now. :\ (But for the joined inputs nothing should change – the icon-confirm should be floating on the right of the input, without background so you don’t even see it’s round.)

Yes, but the input also have a focus and active state, so this is not exactly like that :)
Let's see later how it looks and how we can improve if needed ;)

**Comment by jancborchardt** [comment]
@skjnldsv @ChristophWurst so, could you approve? :)

**Review by karlitschek — APPROVED** [review]

File: core/css/guest.css [hunk 1]
```diff
@@ -575,7 +582,7 @@ form #selectDbType label.ui-state-active {
 	color: #fff;
 	text-align: left;
 	word-wrap: break-word;
-	border-radius: 10px;
+	border-radius: 10px; /* --border-radius-large */
```
Comment (skjnldsv) [inline core/css/guest.css]: :+1: nice with the coments ;)

**Review by skjnldsv — APPROVED** [review]
The inline input + confirm has still the proper design :)
Let's get this in!
capture d ecran_2018-10-23_12-38-04

**Comment by skjnldsv** [comment]
Rebased!

File: core/css/guest.css (hunk 1 above)
Comment (jancborchardt) [reply core/css/guest.css]: Yeah, thought we make the .css forward-compatible and easily searchable for future standardization. :)

**Comment by danxuliu** [comment]
>  The inline input + confirm has still the proper design :)

Not everywhere, although this may be a problem specific to the share authentication page:
round-button-share-auth

**Comment by skjnldsv** [comment]
@danxuliu nice catch!! :)
I'm guessing something from the guest css?

**Review by skjnldsv — DISMISSED** [review]
See https://github.com/nextcloud/server/pull/11943#issuecomment-432217463

**Comment by jancborchardt** [comment]
@danxuliu @skjnldsv fixed :)
screenshot from 2018-10-23 15-01-03

**Comment by jancborchardt** [comment]
Failures seem to be unrelated as far as I can see?
https://drone.nextcloud.com/nextcloud/server/11726/279
https://drone.nextcloud.com/nextcloud/server/11726/282

**Review by skjnldsv — APPROVED** [review]

**Comment by MorrisJobke** [comment]
JSUnit failure is due to codecov issue.

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**